src/System.Memory/src/System/Runtime/InteropServices/SequenceMarshal.cs

```diff
@@ -50,5 +50,14 @@ public static bool TryGetReadOnlyMemory<T>(ReadOnlySequence<T> sequence, out Rea
         {
             return sequence.TryGetReadOnlyMemory(out readOnlyMemory);
         }
+
+        /// <summary>
+        /// Get <see cref="string"/> from the underlying <see cref="ReadOnlySequence{T}"/>.
+        /// If unable to get the <see cref="string"/>, return false.
+        /// </summary>
+        internal static bool TryGetString(ReadOnlySequence<char> sequence, out string text, out int start, out int length)
```

Comment:
Haven't written any test for this an marked it as `internal` as no api approved for it.